tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -496,6 +504,9 @@ ROOT::Experimental::RNTupleDescriptor::CreateModel(const RCreateModelOptions &op
    auto model = RNTupleModel::Create(std::move(fieldZero));
    for (const auto &topDesc : GetTopLevelFields()) {
       auto field = topDesc.CreateField(*this);
+      if (!options.fForwardCompatible && dynamic_cast<RInvalidField *>(field.get())) {
+         throw RException(R__FAIL("field \"" + field->GetQualifiedFieldName() + "\" could not be reconstructed"));
```

Comment:
I think we should hint here that the message/exception can be ignored by setting a flag in a `RCreateModelOptions`, etc.

Reply:
(unless we should the reverse default and only throw when explicitly requested).